src/donate.js

```diff
@@ -176,7 +252,8 @@ window.WebMonetizationScripts.createDonateWidget = function (donation) {
 window.WebMonetizationScripts.donate = function ({
   paymentPointer,
   noRetry,
-  noWidget
+  noWidget,
+  addAdvert
```

Comment:
We might want to clarify what this means by calling it `addCoilAdvert`. Otherwise people might think that it enables/disables their existing ads on their page based on web monetization.